docs: update doc with note (#41)
ceph/persistent-volume-migrator: README.md

   1. `wget https://github.com/ceph/persistent-volume-migrator/releases/download/v0.1.0-alpha/pv-migrator`
4. Run the command to [migrate the PVC(s)](#migrate-the-pvcs)

> **NOTE**: Source and destination storageClass should be using same pool.

Reviewer: ```suggestion
**NOTE**: source and destination StorageClass should use the same pool.
```

Author: changed
